package.json

```diff
@@ -9,8 +9,12 @@
   "scripts": {
     "serve": "vue-cli-service serve --open --skip-plugins eslint",
     "build": "rollup -c",
-    "test:e2e": "vue-cli-service test:e2e",
-    "test:e2e:ci": "vue-cli-service test:e2e --headless --browser chrome",
+    "build:start": "concurrently \"rollup -c -w\" \"npm start\"",
```

Comment:
For the tests to be run on jenkins you would need to add a `test` script that invokes the target script, for example: `"test": "npm run test:e2e:ci"`